### DISCUSSION TITLE
UI Macro - script tag encoding on ServiceNow - Utah

### DISCUSSION THREAD

**Discussion by fiztech-code** [arnoudkooi/SN-Utils#446] [General]
please see screenshots
![image](https://github.com/arnoudkooi/ServiceNow-Utils/assets/66762914/5ff1765a-416e-49ed-9775-959e88093cde)
![image](https://github.com/arnoudkooi/ServiceNow-Utils/assets/66762914/e478e8af-f763-4f1e-ba18-4558cfcfd873)

the script tag gets encoded when saved from VS code..

any ideas how to fix this?
i tried both 'SN Utils' and 'SN Utils OnPrem'

**Comment by arnoudkooi** [comment]
From the [FAQ of sn-scriptsync](https://marketplace.visualstudio.com/items?itemName=arnoudkooicom.sn-scriptsync#faqs)
Hope this will solve it...

I'm using <script> tags in a widget HTML template, and they do not sync to my instance properly. How can I resolve this?
_Solution (See [#24](https://github.com/arnoudkooi/sn-scriptsync/issues/24)): In your ServiceNow instance, set the following system property:_

`glide.rest.sanitize_request_input = false` (This may also apply to UI Macro's and other places where tags get escaped)

**Reply by fiztech-code** [reply]
that worked! Thanks
